### PR TITLE
Feat/selection input icon change

### LINKED ISSUE #127: [Feature] Change input selection icon
**Is your feature request related to a problem? Please describe.**
Someone said that the x icon from the selection input is misleading and it should be changed to something more intuitive.

**Describe the solution you'd like**
The "x" inside the circle needs to go and have only a full circle.

This is the svg path code that needs to be added for the radio input
`M 16 8 A 8 8 0 1 1 0 8 a 8 8 0 0 1 16 0 z z`
And this is for the checkbox input:
`M 2 0 a 2 2 0 0 0 -2 2 v 12 a 2 2 0 0 0 2 2 h 12 a 2 2 0 0 0 2 -2 V 2 a 2 2 0 0 0 -2 -2 H 2 z z`
The the code can be found in components/SelectionInput.tsx

**Additional context**
This is the current input
image

This is how it should look like
image

### PR DESCRIPTION
**Pull request by Dhruvnotfound**
Fixes #127  

- [x] I have read and agree to the [Code of Conduct](https://github.com/eduardconstantin/azure-fundamentals/blob/main/CODE_OF_CONDUCT.md), and have followed the [Contributing guidelines](https://github.com/eduardconstantin/azure-fundamentals/blob/main/CONTRIBUTING.md).

## Pull Request Type

- [x] Other

## Summary
before- showing cross(x) as we select an option

after- showing circle as we select an option
image
Please review the PR! Thank you!

### CODE REVIEW

File: components/QuizExamForm.tsx [hunk 1]
```diff
@@ -318,7 +318,7 @@ const QuizExamForm: React.FC<Props> = ({
                               : "hidden"
                           }`}
                           fillRule="evenodd"
-                          d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM5.354 4.646a.5.5 0 1 0-.708.708L7.293 8l-2.647 2.646a.5.5 0 0 0 .708.708L8 8.707l2.646 2.647a.5.5 0 0 0 .708-.708L8.707 8l2.647-2.646a.5.5 0 0 0-.708-.708L8 7.293 5.354 4.646z"
+                          d="M 2 0 a 2 2 0 0 0 -2 2 v 12 a 2 2 0 0 0 2 2 h 12 a 2 2 0 0 0 2 -2 V 2 a 2 2 0 0 0 -2 -2 H 2 z z"
```
Comment (eduardconstantin) [inline components/QuizExamForm.tsx:321]:
```suggestion
                          d="M 16 8 A 8 8 0 1 1 0 8 a 8 8 0 0 1 16 0 z z"
```

File: components/SelectionInput.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ const SelectionInput = forwardRef<HTMLInputElement, Props>(
               fillRule="evenodd"
               d={
                 type === "radio"
-                  ? "M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM5.354 4.646a.5.5 0 1 0-.708.708L7.293 8l-2.647 2.646a.5.5 0 0 0 .708.708L8 8.707l2.646 2.647a.5.5 0 0 0 .708-.708L8.707 8l2.647-2.646a.5.5 0 0 0-.708-.708L8 7.293 5.354 4.646z"
+                  ? "M 2 0 a 2 2 0 0 0 -2 2 v 12 a 2 2 0 0 0 2 2 h 12 a 2 2 0 0 0 2 -2 V 2 a 2 2 0 0 0 -2 -2 H 2 z z"
```
Comment (eduardconstantin) [inline components/SelectionInput.tsx:68]:
```suggestion
                  ? "M 16 8 A 8 8 0 1 1 0 8 a 8 8 0 0 1 16 0 z z"
```

File: components/SelectionInput.tsx [hunk 3]
```diff
@@ -65,7 +65,7 @@ const SelectionInput = forwardRef<HTMLInputElement, Props>(
...
                   : "M2 0a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2H2zm3.354 4.646L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 1 1 .708-.708z"
```
Comment (eduardconstantin) [inline components/SelectionInput.tsx:69]:
```suggestion
                  : "M 2 0 a 2 2 0 0 0 -2 2 v 12 a 2 2 0 0 0 2 2 h 12 a 2 2 0 0 0 2 -2 V 2 a 2 2 0 0 0 -2 -2 H 2 z z"
```

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

**Comment by Dhruvnotfound** [comment]
all the requested changes has been made 
thanks for checking the pr request

**Comment by danieldanielecki** [comment]
love this solution!

**Comment by eduardconstantin** [comment]
> love this solution!

I guess my idea with x marking the correct answer is not that intuitive :)

**Review by eduardconstantin — APPROVED** [review]
Perfect! Thank you!

**Pull request merged**